ui/app/insights/src/index.ts

```diff
@@ -30,7 +29,7 @@ export const DefaultAppConfig: AppConfig<RootState> = {
 };
 
 // Try and load the app config from JSON and if anything is found amalgamate it with default
-const configURL = (CONFIG_URL_PREFIX || "") + "/manager_config.json";
+const configURL = (MANAGER_URL || "") + "/api/master/configuration/manager";
```

Comment:
The `getManagerConfig` endpoint can return `null` as a valid response but there is no null check here (previously the fileservlet would have thrown a 404 but I like the server returning a null as you don't see an error in the browser console but you need to make sure you handle that situation.